packages/thumbprint-react/components/Switch/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './index.module.scss';
+
+export const SWITCH_COLOR = {
+    blue: 'blue',
+    green: 'green',
+};
+
+export const SWITCH_SIZES = {
+    small: 'small',
+    large: 'large',
+};
+
+export interface SwitchProps {
+    /**
+     * The `id` is added to the switch as an HTML attribute and passed to the `onChange`
+     * function.
+     */
+    id?: string;
+    /**
+     * Determines if the switch is checked.
+     */
+    isChecked?: boolean;
+    /**
+     * Function that runs when a switch value changes. It receives the new boolean value,
+     * and the provided `id` as such:
+     * `props.onChange(event.target.checked, props.id)`.
+     */
+    onChange: (isChecked: boolean, id: string) => void;
+    /**
+     * Disables the component.
+     */
+    isDisabled?: boolean;
+    /**
+     * Determines how the switch sizes. The default size is `small`.
+     */
+    size?: keyof typeof SWITCH_SIZES;
+    /**
+     * This adds an `aria-label` to the element.
+     * the importance of using labels](https://dequeuniversity.com/rules/axe/4.3/select-name).
+     */
+    accessibilityLabel?: string;
+    /**
+     * Determines how the switch colors. The default color is `blue`.
+     */
+    color?: keyof typeof SWITCH_COLOR;
+    /**
+     * A selector hook into the React component for use in automated testing environments.
+     */
+    dataTestId?: string;
+    /**
+     * Switches on a page with the same name will be grouped together when sent to the server.
+     */
+    name?: string;
+}
+
+export default function Switch({
+    id = '',
```

Comment:
I don't think there's a benefit to defaulting to an empty string here. We don't do that for `name`, for example.